src/content/ru/credits.mdoc

```diff
@@ -0,0 +1,41 @@
+### Credits
+
+Сделано вручную с ❤️ от [{% $config.site.author.name.full %}]. Если вы найдете этот инструмент полезным, пожалуйста, рассмотрите возможность [стать моим спонсором], чтобы поддержать его разработку.
+
+Этот проект лицензируется по [лицензии MIT], а исходный код доступен на [GitHub]. Если вы обнаружите какие-либо ошибки, пожалуйста, [сообщите нам об этом].
+
+Спасибо следующим проектам за то, что сделали {% $config.site.title %} возможным:
+- **[iMockup.pro]** для создания изображения макета iPhone для [Mergist] (**лицензия:** [iMockup.pro basic])
+- **[Lucide]** за иконки с открытым исходным кодом (**лицензия:** [ISC и MIT])
+- **[Nawsome]** за их [анимированный загрузчик CSS] (**лицензия:** [MIT])
+- **[Astro Badges]** за их badge
+- За цвета тем:
+	- **[OKSolar]** (**лицензия:** [CC0](https://creativecommons.org/publicdomain/zero/1.0/))
+	- **[gruvbox]** (**лицензия:** [MIT])
+	- **[Catppuccin]** (**лицензия:** [MIT](https://catppuccin.com/licensing/))
+	- **[Dracula]** (**лицензия:** [MIT](https://github.com/dracula/dracula-theme/blob/main/LICENSE))
+	- **[Nord]** (**лицензия:** [MIT](https://github.com/nordtheme/nord/blob/develop/license))
+
+[![Built with Astro](https://astro.badg.es/v2/built-with-astro/small.svg)](https://astro.build)
```

Comment:
nitpick: `alt` text (i.e. `Built with Astro`) can be translated as well. I missed this in a couple places myself so I won't make you update it but pointing it out if you want to while we're here